lib/base/config_reg.c

```diff
@@ -612,38 +609,51 @@ load_config_from_regpath(krb5_context context,
  * Load configuration from registry
  *
  * The registry keys 'HKCU\Software\Heimdal' and
- * 'HKLM\Software\Heimdal' are treated as krb5.conf files.  Each
+ * 'HKLM\Software\Heimdal' are treated as heim.conf files.  Each
```

Comment:
is `heim.conf` a thing or this is a search/replace error?

Reply:
Yup, it's one search/replace too many!  Good catch!